qemu/tests/cfg/cpu_model_negative.cfg

```diff
@@ -9,7 +10,13 @@
             warning_msg = ".${enforce_flag}' not found"
         - smp_lt_maxcpu:
             wrong_cmd = "-smp 8,maxcpus=4,cores=2,threads=2,dies=1,sockets=2"
-            warning_msg = 'maxcpus must be equal to or greater than smp'
+            variants:
+                - old_msg:
```

Comment:
Hum, how to say. After this part is added, always one test case will be skipped, but they are the same scenario.
So, do we have other solutions to fix the warning_msg? e.g:
```
Host_RHEL.x.x.x:
    warning_msg = 'maxcpus must be equal to or greater than smp'
Host_RHEL.y.y.y:
    warning_msg = 'qemu-kvm: Invalid CPU topology: product of the hierarchy must match maxcpus'
```
or
```
cfg:
    old_warning_msg = 'maxcpus must be equal to or greater than smp'
    new_warning_msg = 'qemu-kvm: Invalid CPU topology: product of the hierarchy must match maxcpus'
---
py:
        qemu_version = utils_qemu.get_qemu_version(qemu_binary)[0]
        required_qemu_version = self.params["required_qemu_version"]
        if qemu_version in VersionInterval('[, 6.2.0)'):
            warning_msg = old_warning_msg
        else:
            warning_msg = new_warning_msg
```

Reply:
@PaulYuuu thanks and update, please help to review.